Documents/Pseudocode/Pseudocode_Final

```diff
@@ -58,13 +58,13 @@ Class Network
 		vector<matrix> sum_nabla_b;
 		vector<matrix> sum_nabla_w;
 		vector<int> test_data_indices;
-		vector<int> mini_batch_indices;		//only used in train(), why not limit its scope to that function?
-		int num_correct;			//only used in train(), why not limit its scope to that function?
-		string training_data_filename;		//consider truncating "filename". unnecessary and unwieldy
+		vector<int> mini_batch_indices;		//only used in train(), why not limit its scope to that function? //I agree [-Ami]
+		int num_correct;			//only used in train(), why not limit its scope to that function? //Yes [-Ami]
+		string training_data;		//consider truncating "filename". unnecessary and unwieldy
 		ifstream training_data_infile
 		string expected_values_filename;	//	adding "filename" also makes code easier to read and understand
 		ifstream expected_values_infile		//	when you have other related data structures to specify (-Eli)
-		
+							//	ifstream will work specifically for the files, have no idea for the other data structures [-Ami]	
```

Comment:
I figured that the ofstream object would only be used in createNetworkFile(), so it didn't need to be a class member, but if it improves readability I'd add it to the class, if not only in Pseudocode.cpp

Reply:
Also, what other structures did you mean?

Reply:
By the other structures, I meant the image file, that we will be using later!

Reply:
Very true... we may end up needing to turn this into a template class or something. Afaik, images can still be opened with fstream objects in binary mode, but it'll be a huge PITA to write (though instructive!). We'll probably have to end up making this into a class template or something...

Reply:
Do you think we should ask Slater about this?

Reply:
At some point, yes